TegelSpec.md

```diff
@@ -14,15 +14,15 @@ _EIS_ Gebruik [[Mapbox-Vector-Tile-Specification]] versie 2.1 (January 19th, 201
 In het kort gaat deze specificatie uit van:
 
 * Google Protobuf encoding ([[PBF]])
-* _Winding order_ conform OGC
+* _Winding order_ conform Mapbox Vectortile Specificatie ([[Mapbox-Vector-Tile-Specification]])
 * [OGC valide](https://www.ogc.org/standards/sfa) geometriëen
 
 In deze praktijkrichtlijn wijken wij af van de beschreven bestand extensie `.mvt` zoals genoemd in [2.1 File extension](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#21-file-extension) van de Mapbox specificatie. Ook wijken wij af van de paragraaf [2.2 Multipurpose Internet Mail Extensions (MIME)](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#22-multipurpose-internet-mail-extensions-mime).
 
 ### Eis: Vrije Bestands Extensie
 
 <div class="advisement">
-_EIS_ De bestands extensie voor de vector tiles is vrij in te vullen, alsmede het MIME type.
+_EIS_ De bestandsextensie voor vectortiles is vrij in te vullen. Het Media Type van vectortiles volgens mapbox specificatie moet altijd `application/vnd.mapbox-vector-tile` zijn.
```

Comment:
```suggestion
_EIS_ De bestandsextensie voor vectortiles is vrij in te vullen. Het Media Type van vectortiles moet volgens mapbox specificatie altijd `application/vnd.mapbox-vector-tile` zijn.
```
_dient_ zou eigenlijk nog mooier zijn, maar de verdere tekst is te informeel ;-)